commands/version/index.js

```diff
@@ -450,9 +461,15 @@ class VersionCommand extends Command {
           changelogPreset,
           rootPath,
           tagPrefix: this.tagPrefix,
-        }).then(changelogLocation => {
+        }).then(({ logPath, newEntry }) => {
           // commit the updated changelog
-          changedFiles.add(changelogLocation);
+          changedFiles.add(logPath);
+
+          // add release notes
+          this.releaseNotes.push({
```

Comment:
This `push` needs to be guarded to only execute during `--independent` publish, otherwise "fixed" mode will be making too many releases (each package `CHANGELOG.md` is still updated with package-specific changes in fixed mode, but that isn't meant to be a separate release).

```js
if (independentVersions) {
  this.releaseNotes.push({
    name: pkg.name,
    notes: newEntry,
  });
}
```

The subsequent root push is still okay, that never runs in independent mode. And then the final loop to create github releases will be correct.

Reply:
Instead of wrapping in independent mode, maybe we include all the release notes from each package as a single GH release?

Reply:
Well, coincidentally, that's been a feature request from some users of independent mode, but the current output of the changelog generator isn't quite clean enough (imo) for that to make sense. Also, since independent mode creates tags for each individual package, that means the GitHub tags/releases page gets spammed with a bunch of empty tags (now).

I'm not sure where we would put the "rolled-up independent" changelog. Much less what to call it. But the work you're doing lays the groundwork for future improvements in this area!